tests/core/creators/manual/resizewithcssunits.html

```diff
@@ -0,0 +1,65 @@
+<style>
+	.input-container > div {
+		margin-bottom: 15px;
+	}
+
+	label {
+		display: block;
+	}
+
+	[type="checkbox"] + label {
+		display: inline-block;
+	}
+
+	button {
+		margin-bottom: 15px;
+	}
+</style>
+
+<form class="input-container">
```

Comment:
There is no need to create a form for this simple manual test and fight with whole page refreshing on IE. We are not submitting anything here, so it may be better to just use JS.